README.md

```diff
@@ -7,24 +7,31 @@
 
 The most popular tool for teams to coordinate mapping on OpenStreetMap. With this web application an area of interest can be defined and divided up into smaller tasks that can be completed rapidly. It shows which areas need to be mapped and which areas need a review for quality assurance. You can see the tool in action: log into the widely used [HOT Tasking Manager](https://tasks.hotosm.org/) and start mapping.
 
-[<img src="./docs-old/assets/project-view.gif" />](./docs-old/assets/project-view.gif)
-
 This is Free and Open Source Software. You are welcome to use the code and set up your own instance. The Tasking Manager has been initially designed and built by and for the [Humanitarian OpenStreetMap Team](https://www.hotosm.org/), and is nowadays used by many communities and organizations.
 
 ## Get involved!
 
-* Start by reading our [Code of conduct](./docs-old/code_of_conduct.md)
-* Get familiar with our [contributor guidelines](./docs-old/contributing.md) explaining the different ways in which you can support this project! We need your help!
+* Start by reading our [Code of conduct](https://www.hotosm.org/code-of-conduct)
+* Get familiar with our [contributor guidelines](./docs/developers/contributing-guidelines.md) explaining the different ways in which you can support this project! We need your help!
 * Join the Tasking Manager Collective Meet up- an opportunity to meet other Tasking Manager contributors. The meet ups take place on the first Wednesday of the month 9:00 or 15:00UTC! Register to receive a calendar invite: https://bit.ly/3s6ntmV or join directly via this link: https://meet.jit.si/TaskingManagerCollectiveMeetUp
 
 
 ## Developers
 
-* [Install TM with Docker](./docs-old/setup-docker.md)
-* [Setup the TM for development](./docs-old/setup-development.md)
-* [Learn about migrations between major versions](./docs-old/migration.md)
+* [Understand the code](./docs/developers/understanding-the-code.md)
+* [Setup the TM for development](./docs/developers/development-setup.md)
+* [Learn about vesions and releases](./docs/developers/versions-and-releases.md)
 * Help us and submit [pull requests](https://github.com/hotosm/tasking-manager/pulls)
 
+## Sysadmins
+
+* [architecture](./docs/sysadmins/architecture.md)
+* [ci-cd](./docs/sysadmins/ci-cd.md)
+* [deployment](./docs/sysadmins/deployment.md)
+* [monitoring-logging](./docs/sysadmins/monitoring-logging.md)
+* [networking-connectivity](./docs/sysadmins/networking-connectivity.md)
+* [security-reliability-processes](./docs/sysadmins/security-reliability-processes.md)
```

Comment:
Can we include this section in the `Setup the TM for development` document rather than the Readme file?